test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -140,7 +136,7 @@ end
             end
             @test X̄.U ≈ ones(3, 2) atol=1e-6
             @test X̄.S ≈ ones(2) atol=1e-6
-            @test X̄.V ≈ ones(2, 2) atol=1e-6
+            @test X̄.Vt ≈ 2 * ones(2, 2) atol=1e-6 # * 2 because V and Vt both accumulate to Vt
```

Comment:
Could you move these tests into the loop to avoid this collision of two cotangents?

Reply:
Then we end up with something like
`@test getproperty(dF, p) == ones(size(getproperty(X̄, p)))`
because we have to take care of the sizes. And that's probably less clear?